Dockerfiles/agent/s6-services/network/finish

```diff
@@ -5,10 +5,10 @@
 ifthenelse
     { s6-test ${1} -eq 0 }
     {
-        foreground { /initlog.sh "network-tracer exited with code ${1}, disabling" }
+        foreground { /initlog.sh "system-probe exited with code ${1}, disabling" }
```

Comment:
Does it make sense to move these to a different directory? e.g. `Dockerfiles/agent/s6-services/system-probe/*`